### PR TITLE
Add support for angular based multi-line placeholder for text-area

### PR DESCRIPTION
**Pull request by Chiranjivee**

### CODE REVIEW

File: angularjs-version/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    let scrollTop = 0;
+    $scope.mouseWheelHandler = function (myEvent) {
+        myEvent.deltaY < 0 ?
+            document.getElementById("uniq").scrollTop = scrollTop -= 10:
+            document.getElementById("uniq").scrollTop = scrollTop += 10;
+    };
+})
+.directive("ngWheel", ["$parse", function($parse) {
+    return function(scope, element, attr) {
+        var fn = $parse(attr.ngWheel);
```
Comment (dgr8akki) [inline angularjs-version/index.js]: Please use **let/const** instead of var

File: angularjs-version/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        }
+
+        let length = myEvent.target.value.length;
+        length > 0 ?
+            myEvent.target.classList.add("data-edits"):
+            myEvent.target.classList.remove("data-edits");
+    };
+
+    let scrollTop = 0;
+    $scope.mouseWheelHandler = function (myEvent) {
```
Comment (dgr8akki) [inline angularjs-version/index.js]: Try to use arrow functions instead.

File: angularjs-version/index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<html>
+    <head>
+        <link rel="stylesheet" href="index.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.9/angular.min.js"></script>
```
Comment (dgr8akki) [inline angularjs-version/index.html]: Put scripts in the footer tag.

File: angularjs-version/index.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<html>
+    <head>
```
Comment (dgr8akki) [inline angularjs-version/index.html:2]: Title is missing in header

**Review by dgr8akki — CHANGES_REQUESTED** [review]

File: angularjs-version/index.js (hunk 1 above)
Comment (Chiranjivee) [reply angularjs-version/index.js]: Updated

File: angularjs-version/index.js (hunk 2 above)
Comment (Chiranjivee) [reply angularjs-version/index.js]: Updated

File: angularjs-version/index.html (hunk 3 above)
Comment (Chiranjivee) [reply angularjs-version/index.html]: Updated

File: angularjs-version/index.html (hunk 4 above)
Comment (Chiranjivee) [reply angularjs-version/index.html:2]: Updated

**Pull request merged**